samples/sp_whoIsActive/package.json

```diff
@@ -4,10 +4,12 @@
     "description": "sp_whoisactive for SQL Operations Studio",
     "version": "0.1.0",
     "publisher": "Microsoft",
+    "preview": true,
     "engines": {
-        "vscode": "^1.19.0",
+        "vscode": "0.10.x",
```

Comment:
why are we using the vscode version here and a different on in the other extension?